subsys/bluetooth/controller/hci/hci.c

```diff
@@ -6548,6 +6549,18 @@ static void le_per_adv_sync_report(struct pdu_data *pdu_data,
 		return;
 	}
 
+	/* FIXME: make node_rx self sufficient with data when generating so that
```

Comment:
Is that possible? How to e.g. stop reports if the `sync` is terminated by Host after `node_rx` is enqueued towards ULL/HCI?

Reply:
ok, then it is possible that when this code is executed the `sync` is already released. Are you sure that it will never be overwritten between the release and 'sync->timeout_reload` access, that is done here?

If there is no other way to solve this, there should be an information that `timeout_reload` may not be moved from safe position. You could use `BUILD_ASSERT` and verify if the `timeout_reload` is at fixed offset from beginning of the `lll_sync`